### DISCUSSION TITLE
Merged Github account

### DISCUSSION THREAD

**Discussion by FreiLoris** [vercel/community#6053] [Account Issue] [answered]
Hey I merged yesterday my two github accounts. I had to ship every repo to my new account and afterwards I deleted my github account, so that I have only one account left with every repository.
Because I use github to logged in on vercel, I couldn't log in because the github account does not exist anymore... Now my projects are still running on the old vercel account as you see:
https://nadine-beige.vercel.app/
Is there a possibility that you could transfer the two projects (nadine-backend and nadine) to my new vercel account or just delete the old account? The old github account I logged in to vercel was: ****** (I dont want to share it public).
If you need more information just text me.

**Comment by amyegan** [comment, marked as answer]
Hey @FreiLoris. It sounds like you deleted the GitHub account but not the Vercel account. In that case, you should still be able to log in using the email address for that account.
